frontend/src/pages/archive/crawl-templates-detail.ts

```diff
@@ -1029,15 +1038,14 @@ export class CrawlTemplatesDetail extends LiteElement {
   private async duplicateConfig() {
     if (!this.crawlTemplate) return;
 
-    const config: CrawlTemplate["config"] = {
-      ...this.crawlTemplate.config,
+    const crawlTemplate: InitialCrawlTemplate = {
+      name: msg(str`${this.crawlTemplate.name} Copy`),
+      config: this.crawlTemplate.config,
+      profileid: this.crawlTemplate.profileid || "",
```

Comment:
```suggestion
      profileid: this.crawlTemplate.profileid,
```
Just set the profileid to null when no profile